deploy/compose.yaml

```diff
@@ -18,7 +18,9 @@ services:
     ports:
       - "<host-port>:8078"
     volumes:
-      - "recipya-data:/root/.config/Recipya"
+      - "recipya-data:/app/.config/Recipya"
```

Comment:
I am unsure about the volume path here... would it then really be `/app/...` instead of `/root/...`?

I couldn't test the changes locally with this `compose.yaml` file. Somehow the Dockerfile didn't build even after adding this (it's not pushed to the PR):

```yaml
services:
  web:
    build:
      context: .
```

Reply:
I'm also unsure about the path. I'll look into it once I've got a few.

Reply:
You may test locally with the following change to your `compose.yaml`:
```Dockerfile
services:
  web:
    build:
      context: ..
      dockerfile: deploy/Dockerfile
    ...
```

However, the following error is thrown once the app is started:
```text
recipya  | panic: mkdir /.config: permission denied
recipya  | 
recipya  | goroutine 1 [running]:                                                                                                                                
recipya  | github.com/reaper47/recipya/internal/app.Init()
recipya  |      /app/internal/app/app.go:274 +0x80c  
```

Caused by:
```go
// internal/app/app.go
func Init() {
        dir, err := os.UserConfigDir()
	if err != nil {
		panic(err)
	}
	baseDir := filepath.Join(dir, "Recipya")

	BackupPath = filepath.Join(baseDir, "Backup")
	DBBasePath = filepath.Join(baseDir, "Database")
	ImagesDir = filepath.Join(baseDir, "Images")
	LogsDir = filepath.Join(baseDir, "Logs")
	ThumbnailsDir = filepath.Join(ImagesDir, "Thumbnails")
	VideosDir = filepath.Join(baseDir, "Videos")

	xs := []string{BackupPath, DBBasePath, ImagesDir, ThumbnailsDir, LogsDir, VideosDir}
	for _, s := range xs {
		err = os.MkdirAll(s, os.ModePerm)
		if err != nil {
			panic(err) // <-- here
		}
	}
        ...
}
```

I'm not sure how to solve this one.